index.test-d.ts

```diff
@@ -23,5 +23,12 @@ pTimeout(delayedPromise(), 50, () => 10).then(value => {
 	expectType<string | number>(value);
 });
 
+pTimeout(delayedPromise(), 50, undefined, {customTimers: {setTimeout}});
+pTimeout(delayedPromise(), 50, 'foo', {customTimers: {clearTimeout}});
+pTimeout(delayedPromise(), 50, new Error('error'), {customTimers: {setTimeout, clearTimeout}});
+pTimeout(delayedPromise(), 50, () => 10, {});
+
+expectError(pTimeout(delayedPromise(), 50, () => 10, {customTimers: {setTimeout: () => true}}));
```

Comment:
Interestingly, this call to `expectError` only worked after upgrading `tsd` from `0.7.2` to `0.13.1`.